boomi/boomi-workflow.xml

```diff
@@ -0,0 +1,152 @@
+<Workflow name="Qmasters Boomi" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
+    <Parameters>
+        <Parameter name="identifier" label="Boomi" description="The log source identifier to post the events to." required="true" />
+        <Parameter name="account_id" label="Instance Name" description="Boomi account ID." required="true" />
+        <Parameter name="username" label="Username" description="Boomi user name." required="true" />
+        <Parameter name="token" label="Token" description="Boomi user's Token." required="true" secret="true" />
+        <Parameter name="time_zone" label="Time Zone" description="The timezone to be used." default="UTC" />
+        <Parameter name="start_fetch_time" label="Initial Event Start Time Date" description="Date time from which events will be initially retrieved." default="7" />
```

Comment:
```suggestion
        <Parameter name="start_fetch_time" label="Initial Event Start Time Date" description="Date time from which events will be initially retrieved." />
```

we need to add a default time for the first tun in case the user leaves this parameter empty.